locales-pending/dashboard-premium.ftl

```diff
@@ -38,6 +38,15 @@ modal-heres-what-we-fixed-description-part-three = <b>In Progress</b> includes a
 dashboard-exposures-filter-exposure-type = Exposure type
 dashboard-exposures-filter-exposure-type-info-for-sale = Your info for sale
 dashboard-exposures-filter-exposure-type-data-breach = Data breach
+dashboard-exposures-filter-exposure-removal-time-title = Removal time
+dashboard-exposures-filter-exposure-removal-time-label-7 = Up to 7 days
+dashboard-exposures-filter-exposure-removal-time-label-13 = 8–13 days
+dashboard-exposures-filter-exposure-removal-time-label-60 = 14–60 days
+dashboard-exposures-filter-exposure-removal-time-label-90 = 61–90 days
+dashboard-exposures-filter-exposure-removal-time-label-180 = 91-180 days
+dashboard-exposures-filter-exposure-removal-time-label-other = 181+ days
+dashboard-exposures-filter-exposure-removal-time-label-unknown = Unknown
+dashboard-exposures-filter-exposure-removal-time-label-na = N/A
```

Comment:
Not sure if this acronym might be unknown to potential future localisers:

```suggestion
# "Not applicable" - abbreviated because there's not much room for this string.
dashboard-exposures-filter-exposure-removal-time-label-na = N/A
```

Reply:
Updated in [01195cc](https://github.com/mozilla/blurts-server/pull/5189/commits/01195cc51ba40128baee81dcaafc99d000fbfa43).